lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -118,10 +123,23 @@ public APIGatewayProxyResponseEvent handleRequest(
                     jarValidator.validateRequestJwt(
                             signedJWT, sessionParams.get(CLIENT_ID_PARAM_KEY));
 
-            String govukSigninJourneyId = claimsSet.getStringClaim("govuk_signin_journey_id");
-            String emailAddress = claimsSet.getStringClaim("email_address");
+            String govukSigninJourneyId =
+                    claimsSet.getStringClaim(REQUEST_GOV_UK_SIGN_IN_JOURNEY_ID_KEY);
+            String emailAddress = claimsSet.getStringClaim(REQUEST_EMAIL_ADDRESS_KEY);
             LogHelper.attachGovukSigninJourneyIdToLogs(govukSigninJourneyId);
 
+            List<String> vtr = claimsSet.getStringListClaim(REQUEST_VTR_KEY);
+            if (vtr == null || vtr.isEmpty() || vtr.stream().allMatch(String::isEmpty)) {
+                var message =
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        ErrorResponse.MISSING_VTR.getMessage());
+                LOGGER.error(message);
+
+                return ApiGatewayResponseGenerator.proxyJsonResponse(
+                        HttpStatus.SC_BAD_REQUEST, ErrorResponse.MISSING_VTR);
```

Comment:
Ok, I've spoken to orchestration and I think I'd like to do this in two stages, as we did before:
- For now, only log the error message
- Once we're confident it's not breaking anything in production, turn it into a real error

Sorry we didn't capture this as a requirement at the start!